account_invoice_report_grouped_by_picking/models/account_move.py

```diff
@@ -101,45 +91,51 @@ def lines_grouped_by_picking(self):
                     }
                 )
                 continue
+
```

Comment:
Please don't add empty lines inside a method, for both continuing with the existing style and as PEP8 recommendation.

Reply:
Changes done, but PEP8 allow blank lines in this cases:

> Extra blank lines may be used (sparingly) to separate groups of related functions. Blank lines may be omitted between a bunch of related one-liners (e.g. a set of dummy implementations).
> 
> Use blank lines in functions, sparingly, to indicate logical sections.
> 


https://peps.python.org/pep-0008/#blank-lines

Reply:
The main problem is that this method is too long and smaller methods should be created.
I discarded that change because all these methods are being declared in the account.move model.
The ultimate solution is to bring all this logic into an abstract_report, but I didn't want to do that in a stable release.
Let's keep that in mind for the migration to the new version.